hygieia/.openshift/templates/projects.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Can we delete this and point to: https://raw.githubusercontent.com/redhat-cop/cluster-lifecycle/v3.9.0/files/projectrequest/template.yml instead?

need to add `NAMESPACE_DISPLAY_NAME: "{{ namespace }}"` to params if using it.